boot/C2HsLexer.x

```diff
@@ -0,0 +1,90 @@
+{
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
+module Distribution.C2Hs.Lexer ( getImports ) where
+
+import Control.Applicative ((<$>))
+}
+
+%wrapper "monad"
+
+$module = [A-Za-z\.]
+
+tokens :-
```

Comment:
This file needs comments, what it does. Is there a reason why you chose `alex`, and not writing something using `parsec`?

Reply:
I used `alex` because it seems to be the best choice for lexers and I've used it in the past for identifying dependencies in e.g. C.

Reply:
Well, I don't agree that we need to write any new `alex` code. It's write-only for almost any other contributor. You should been asked before making tech choice